src/Microsoft.AspNetCore.Server.HttpSys/NativeInterop/HttpApi.cs

```diff
@@ -70,724 +70,6 @@ internal static unsafe class HttpApi
-                HttpHeaderVia = 8,    // general-header [section 4.5]
-                HttpHeaderWarning = 9,    // general-header [section 4.5]
-
-                HttpHeaderAllow = 10,   // entity-header  [section 7.1]
-                HttpHeaderContentLength = 11,   // entity-header  [section 7.1]
-                HttpHeaderContentType = 12,   // entity-header  [section 7.1]
-                HttpHeaderContentEncoding = 13,   // entity-header  [section 7.1]
-                HttpHeaderContentLanguage = 14,   // entity-header  [section 7.1]
-                HttpHeaderContentLocation = 15,   // entity-header  [section 7.1]
-                HttpHeaderContentMd5 = 16,   // entity-header  [section 7.1]
-                HttpHeaderContentRange = 17,   // entity-header  [section 7.1]
-                HttpHeaderExpires = 18,   // entity-header  [section 7.1]
-                HttpHeaderLastModified = 19,   // entity-header  [section 7.1]
-
-                // Response Headers
-
-                HttpHeaderAcceptRanges = 20,   // response-header [section 6.2]
-                HttpHeaderAge = 21,   // response-header [section 6.2]
-                HttpHeaderEtag = 22,   // response-header [section 6.2]
-                HttpHeaderLocation = 23,   // response-header [section 6.2]
-                HttpHeaderProxyAuthenticate = 24,   // response-header [section 6.2]
-                HttpHeaderRetryAfter = 25,   // response-header [section 6.2]
-                HttpHeaderServer = 26,   // response-header [section 6.2]
-                HttpHeaderSetCookie = 27,   // response-header [not in rfc]
-                HttpHeaderVary = 28,   // response-header [section 6.2]
-                HttpHeaderWwwAuthenticate = 29,   // response-header [section 6.2]
-
-                HttpHeaderResponseMaximum = 30,
-
-                HttpHeaderMaximum = 41
-            }
-        }
-
-        private static HTTPAPI_VERSION version;
-
-        // This property is used by HttpListener to pass the version structure to the native layer in API
-        // calls. 
-
-        internal static HTTPAPI_VERSION Version
-        {
-            get
-            {
-                return version;
-            }
-        }
-
-        // This property is used by HttpListener to get the Api version in use so that it uses appropriate 
-        // Http APIs.
-
-        internal static HTTP_API_VERSION ApiVersion
-        {
-            get
-            {
-                if (version.HttpApiMajorVersion == 2 && version.HttpApiMinorVersion == 0)
-                {
-                    return HTTP_API_VERSION.Version20;
-                }
-                else if (version.HttpApiMajorVersion == 1 && version.HttpApiMinorVersion == 0)
-                {
-                    return HTTP_API_VERSION.Version10;
-                }
-                else
-                {
-                    return HTTP_API_VERSION.Invalid;
-                }
-            }
-        }
-
         static HttpApi()
```

Comment:
Let's change this to a static Initialize method so that it becomes optional. HttpSys needs it, IIS doesn't.